src/utils.ts

```diff
@@ -126,3 +126,7 @@ export function distance2d(x1: number, y1: number, x2: number, y2: number) {
   const yd = y2 - y1;
   return Math.sqrt(xd * xd + yd * yd);
 }
+
+export function resetCursor() {
+  document.documentElement.style.cursor = "";
+}
```

Comment:
Since you've moved this, I propose moving `setCursorForShape` from `index.tsx` here, too.

Reply:
...or never mind, too much work (it's coupled to other things). Let's leave it for later.